clojure.html.markdown

```diff
@@ -306,6 +306,7 @@ keymap ; => {:a 1, :b 2, :c 3}
 ;;;;;;;;;;;;;;;
 
 ; Use "use" to get all functions from the module
+; without the need to call functions with its namespace
```

Comment:
Hello @AhmedMozaly - thank you for your contribution.

I'm not quite sure what you are trying to say: Is "functions" supposed to be singular? or do you maybe mean "within" instead of "with"?